### ISSUE TITLE
requestReview() doesn't trigger the rating prompt  in android

### ISSUE THREAD

**Issue by minafa96** [oblador/react-native-store-review#51]
we have put the app in the google play internal test track

and my Gmail is added as a tester

and I've downloaded the app from google play 

and I have not reviewed the app before

but nothing shows up when requestReview() and in the dev mood only the 'available' appears.

```<NormalButton
                onPress={() => {
                  if (StoreReview.isAvailable) {
                    console.warn('available');
                    StoreReview.requestReview();
                  }
                }}
                text="Write your review"
              />
```

"react-native-store-review": "^0.2.0",

"react-native": "0.63.4",

device android version : 9

**Comment by pabloluz** [comment]
@minafa96 I'm experiencing the same issue. Please check https://github.com/oblador/react-native-store-review/issues/48 and have a look at the comment I left there ;)

**Comment by minafa96** [comment]
@pabloluz Thanks for the reply. where should that proguard rules be added exactly?

**Comment by pabloluz** [comment]
@minafa96 there should be a file called `proguard-rules.pro` under your `PROJECT_DIR/android/app` folder. 

On that file, you should see a comment like `# Add any project-specific keep options here:`. Simply add those rules below that comment.

**Comment by minafa96** [comment]
> @minafa96 there should be a file called `proguard-rules.pro` under your `PROJECT_DIR/android/app` folder.
> 
> On that file, you should see a comment like `# Add any project-specific keep options here:`. Simply add those rules below that comment.

Thanks, but no changes :(

by the way, I'm not getting any error or warning at all

**Comment by minafa96** [comment]
If anyone has this problem use internal app sharing instead of internal test track, worked for me ☺️ 🎉

**Comment by salik-a** [comment]
> If anyone has this problem use internal app sharing instead of internal test track, worked for me ☺️ 🎉

what does internal app sharing instead of internal test track means?

**Comment by oblador** [comment]
Fixed in 0.4.1
